tools/build-image.sh

```diff
@@ -131,63 +131,24 @@ fi
 
 set -xe
 
-glassdir=${srcdir}/glass
-graveldir=${srcdir}/gravel
-
-[[ ! -d "${glassdir}" ]] && \
-  error_exit "missing frontend directory at '${glassdir}"
```

Comment:
As a bit of a regression we lose some of these checks. I'm not sure of their value though; if people are deleting things from the source they can't expect it to just build.

Reply:
IMO those checks aren't especially useful; as you say, someone would have had to delete something from the source tree to hit them. If someone does that with the Makefile form, they'll hit an error anyway (e.g. `cd glass` will fail with something like `/bin/sh: line 0: cd: src/glass: No such file or directory`)